src/components/CurrentRewardCycle.js

```diff
@@ -1,23 +1,44 @@
 import { useAtom } from 'jotai';
-import { useEffect } from 'react';
-import { BLOCK_HEIGHT, refreshBlockHeight, refreshRewardCycle, REWARD_CYCLE } from '../lib/blocks';
+import { useEffect, useState } from 'react';
+import {
+  BLOCK_HEIGHT,
+  getFirstBlockInCycle,
+  refreshBlockHeight,
+  refreshRewardCycle,
+  REWARD_CYCLE,
+} from '../lib/blocks';
 import { REWARD_CYCLE_LENGTH } from '../lib/constants';
 
 export function CurrentRewardCycle() {
   const [blockHeight, setBlockHeight] = useAtom(BLOCK_HEIGHT);
-  const [rewardCycle, setRewardCycle] = useAtom(REWARD_CYCLE);
+  const [rewardCycle, setRewardCycle] = useState();
+  const [firstBlockInCycle, setFirstBlockInCycle] = useState();
   const rewardCycleLength = REWARD_CYCLE_LENGTH;
 
   useEffect(() => {
     refreshBlockHeight(setBlockHeight);
   }, [setBlockHeight]);
 
   useEffect(() => {
-    refreshRewardCycle(26600);
-  }, [setRewardCycle]);
+    refreshRewardCycle(blockHeight.value).then(cycle => {
```

Comment:
Correct, good catch! I had changed it away from using an atom and was playing around with a few things - but missed taking that out. With #78 merged I can now look at using atom values for start/end block height instead of what I'm doing here. :pray: